src/components/transactionHistory/HistoryTableRow.tsx

```diff
@@ -182,6 +206,14 @@ const HistoryTableRow: React.FC<{
                   )}
                </div>
             );
+         } else if (isLightningTransaction(tx) && tx.status === TxStatus.PENDING) {
+            return (
+               <div className='flex justify-center'>
+                  <button onClick={() => handleTryToMint(tx)}>
```

Comment:
should this be disabled or something when already reclaiming?

also when it will happen that you need to go to the history page to reclaim?